genesis/utils/usd.py

```diff
@@ -0,0 +1,312 @@
+                                opacity_texture = None
+                                roughness_texture = None
+                                metallic_texture = None
+
+                                if diffuse_texture_path:
+                                    diffuse_texture_path = os.path.normpath(
+                                        os.path.join(os.path.dirname(mdl_asset_path), diffuse_texture_path)
+                                    )
+                                    diffuse_image = mu.PIL_to_array(Image.open(diffuse_texture_path))
+                                    color_texture = mu.create_texture(diffuse_image, diffuse_tint, "srgb")
+                                    gs.logger.info(f"Get Diffuse texture: {diffuse_texture_path}")
+
+                                if normal_texture_path:
+                                    normal_texture_path = os.path.normpath(
+                                        os.path.join(os.path.dirname(mdl_asset_path), normal_texture_path)
+                                    )
+                                    normal_image = mu.PIL_to_array(Image.open(normal_texture_path))
+                                    normal_texture = mu.create_texture(normal_image, None, "linear")
+                                    gs.logger.info(f"Get Normal texture: {normal_texture_path}")
+
+                                if orm_texture_path:
+                                    orm_texture_path = os.path.normpath(
+                                        os.path.join(os.path.dirname(mdl_asset_path), orm_texture_path)
+                                    )
+                                    orm_image = mu.PIL_to_array(Image.open(orm_texture_path))
+                                    # Split ORM image into separate textures
+                                    opacity_texture = mu.create_texture(orm_image[:, :, 0], None, "linear")
+                                    roughness_texture = mu.create_texture(orm_image[:, :, 1], None, "linear")
+                                    metallic_texture = mu.create_texture(orm_image[:, :, 2], None, "linear")
+                                    gs.logger.info(f"Get ORM texture: {orm_texture_path}")
+
+                                mesh_surface.update_texture(
+                                    color_texture=color_texture,
+                                    normal_texture=normal_texture,
+                                    opacity_texture=opacity_texture,
+                                    roughness_texture=roughness_texture,
+                                    metallic_texture=metallic_texture,
+                                )
+
+            meshes.append(
+                gs.Mesh.from_attrs(
+                    verts=verts,
+                    faces=faces,
+                    normals=normals,
+                    surface=mesh_surface,
+                    uvs=uvs,
+                    scale=scale,
+                )
+            )
+
+    del stage
```

Comment:
Useless something is terribly wrong with `pxr` lib, there is no need to call `del` manually since the variable is getting out of scope. It will be garbage collected and deleted automatically.